docs-guides/source/convert-a-pytorch-segmentation-model.md

```diff
@@ -140,46 +134,57 @@ class WrappedDeeplabv3Resnet101(nn.Module):
     def __init__(self):
         super(WrappedDeeplabv3Resnet101, self).__init__()
         self.model = torch.hub.load(
-            'pytorch/vision:v0.6.0',
-            'deeplabv3_resnet101',
+            "pytorch/vision:v0.6.0",
+            "deeplabv3_resnet101",
             pretrained=True
         ).eval()
 
-    def forward(self, x):
-        res = self.model(x)
+    def forward(self, input):
+        res = self.model(input)
         # Extract the tensor we want from the output dictionary
         x = res["out"]
         return x
 ```
 
-Now the trace runs without errors:
+Now the trace / export runs without errors:
 
 ```python
-traceable_model = WrappedDeeplabv3Resnet101().eval()
-trace = torch.jit.trace(traceable_model, input_batch)
+torch_model = WrappedDeeplabv3Resnet101().eval()
+traced_model = torch.jit.trace(torch_model, input_batch)
+exported_program = torch.export.export(torch_model, (input_batch,))
 ```
 
 ## Convert the Model
 
 Follow these steps:
 
-1. Pass in the traced model to [`convert()`](https://apple.github.io/coremltools/source/coremltools.converters.convert.html#module-coremltools.converters._converters_entry) to produce a Core ML model (ML program), and include the inputs to provide to the model:
+1. Pass in the traced / exported model to [`convert()`](https://apple.github.io/coremltools/source/coremltools.converters.convert.html#module-coremltools.converters._converters_entry) to produce a Core ML model (ML program), and include the inputs to provide to the model:
     
     ```python
-    mlmodel = ct.convert(
-        trace,
+    mlmode_from_trace = ct.convert(
+        traced_model,
         inputs=[ct.TensorType(name="input", shape=input_batch.shape)],
     )
     ```
-    
+
+    ``` {note}
+    This example includes a name for the input to make it easier to construct the Core ML model's input dictionary. To learn more about input options, see [Flexible Input Shapes](flexible-inputs).
+    ```
+
+    or
+
+    ```python
+    mlmode_from_export = ct.convert(exported_program)
+    ```
+
     ``` {note}
-    This example includes a name for the output to make it easier to extract from the Core ML model's prediction dictionary. To learn more about input options, see [Flexible Input Shapes](flexible-inputs).
+    For torch.export, the input name is inherited from the torch model
```

Comment:
What if the user wants to provide a custom name themselves and not inherit the name that's in the Torch model? Is there a way to do that?

Reply:
Unfortunately that's not supported yet. Added a sentence
```
As of Core ML Tools 8.0, the input name cannot be customized yet
```